package.json

```diff
@@ -1,31 +1,40 @@
 {
   "name": "nuka-carousel",
-  "version": "3.0.1",
+  "version": "4.0.0",
   "description": "Pure React Carousel",
   "main": "index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
   "dependencies": {
     "create-react-class": "^15.6.2",
+    "cross-env": "^5.1.3",
     "exenv": "^1.2.0",
-    "react-tween-state": "^0.1.5",
     "object-assign": "^4.1.0",
-    "prop-types": "^15.6.0"
+    "prop-types": "^15.6.0",
+    "react-tween-state": "^0.1.5",
+    "webpack-cli": "^2.0.9"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
+    "@babel/core": "^7.0.0-beta.40",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.40",
+    "@babel/polyfill": "^7.0.0-beta.40",
+    "@babel/preset-env": "^7.0.0-beta.40",
+    "@babel/preset-react": "^7.0.0-beta.40",
+    "@babel/cli": "^7.0.0-beta.40",
     "babel-eslint": "^7.2.3",
-    "babel-loader": "^6.4.1",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-react": "^6.24.1",
-    "babel-register": "^6.26.0",
+    "babel-loader": "8.0.0-beta.2",
+    "builder": "^4.0.0",
     "chai": "^3.3.0",
-    "css-loader": "^0.19.0",
+    "css-loader": "^0.28.10",
     "del": "^2.0.2",
     "eslint": "^4.8.0",
-    "eslint-loader": "^1.0.0",
+    "eslint-config-formidable": "^3.0.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-loader": "^2.0.0",
     "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-filenames": "^1.2.0",
+    "eslint-plugin-import": "^2.9.0",
+    "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.4.0",
     "gulp": "^3.9.0",
```

Comment:
Can we remove all the gulp deps and the`Gulpfile`? Or is that part of the tests?

Reply:
Yeah, tests don't work at all anymore. We're going to move to jest. So I can just remove that stuff for now.